packages/repo-tools/src/commands/api-reports/api-extractor.ts

```diff
@@ -840,7 +840,9 @@ export async function buildDocs({
           context.writer.writeLine('---');
           for (const [name, value] of Object.entries(node.values)) {
             if (value) {
-              context.writer.writeLine(`${name}: ${value}`);
+              context.writer.writeLine(
+                `${name}: "${String(value).replace(/\"/g, '')}"`,
```

Comment:
Should they be kept but escaped `\"` instead?

Reply:
I don't think so, this is what it looks like today 😅 
<img width="713" alt="Screenshot 2022-12-15 at 16 31 50" src="https://user-images.githubusercontent.com/68980/207901404-bf31f770-97ff-4bb5-8c66-c991ace276b8.png">

Reply:
Aha! Then no. :)